src/browser/Terminal.ts

```diff
@@ -1134,6 +1134,15 @@ export class Terminal extends CoreTerminal implements ITerminal {
       return true;
     }
 
+    if (event.key && !event.ctrlKey && !event.altKey && !event.metaKey && event.key.length === 1 && (
+      (event.keyCode >= 65 && event.keyCode <= 90) ||
+      (event.keyCode >= 97 && event.keyCode <= 122)
+    )) {
+      // Include only keys in A-Z/a-z.
+      // HACK: Need process these key events in 'onkeypress' to fix a IME bug on macOS Chrome. fix for #2457
+      return true;
```

Comment:
If concern about event.key is non-English char, I think it can also add a condition to confirm the event.key is a-zA-Z before returning.

```suggestion
      const c = event.key.charCodeAt(0)
      if ((c >= 65 && c <= 90) || (c >= 97 && c <= 122)) {
        return true;
      }
```